### ISSUE TITLE
Using Meikyo does not highlight GCDs to indicate that weaponskills can be executed without combo prerequisites.

### ISSUE THREAD

**Issue by jj-xiv** [UnknownX7/NoClippy#3]
![1wngiUw](https://user-images.githubusercontent.com/93424697/139525443-8450b377-a8f5-4dd6-8c44-f62c58aa4f93.png)
First picture. NoClippy enabled.
Meikyo - 3 charges. None of the weaponskills are highlighted.

Second picture. NoClippy disabled. It SHOULD look like this:
![DrEcO7W](https://user-images.githubusercontent.com/93424697/139525475-dccf5928-1ed2-45d6-a166-40ba9848be5b.png)

**Comment by UnknownX7** [comment]
I am unable to reproduce this, can you provide a picture of your installed plugins (as well as try testing with only NoClippy if you have others), and NoClippy's settings?

**Comment by jj-xiv** [comment]
Thank you for your response. 

I logged in today, and everything seems to be back to normal. So maybe a simple restart fixed it. 
Just for your information, the only setting on NoClippy I had checked was "Enable Anim. Lock Comp."

If it happens again, I will definitely try testing with only NoClippy enabled.

**Comment by BeetleLord** [comment]
I can confirm that this happened to me while messing with other addons. Restarting seemed to fix.

**Comment by seanryanchan** [comment]
This also happens to me as well. 

Just to add however, the effect of Meikyo does still apply.

**Comment by UnknownX7** [comment]
This may have been fixed with the latest update, please let me know if it's still happening.

**Comment by DAOWAce** [comment]
If that was the update to support Dalamud 7.0, then I still saw it happening during a dungeon run.  Strangely it fixed itself mid run, then broke again.

That said, I found a workaround for my Alexander compatibility issue, so I've reverted back to that myself.  Interestingly running NoClippy together resulted in it spewing a latency error and changing to logging mode instantly after using an ability.  I don't know how they're supposed to work together, but I guess not like this?

As an aside, I also saw this issue with Mudras with the experimental option checked: 
https://user-images.githubusercontent.com/480587/187085185-dd07ae3e-490a-4e96-83ff-856e00e028ad.mp4
